packages/astro/src/@types/astro.ts

```diff
@@ -452,6 +453,53 @@ export interface AstroUserConfig {
 	 */
 	cacheDir?: string;
 
+	
+
+	/**
+	 * @docs
+	 * @name redirects (Experimental)
+	 * @type {RedirectConfig}
+	 * @default `{}`
+	 * @version 2.6.0
+	 * @description Specify a mapping of redirects where the key is the route to match
+	 * and the value is the path to redirect to. 
+	 *
+	 * You can redirect both static and dynamic routes, but only to the same kind of route.
+	 * For example you cannot have a `'/article': '/blog/[...slug]'` redirect.
+	 * 
+	 * 
+	 * ```js
+	 * {
+	 *   redirects: {
+	 *     '/old': '/new',
+	 *     '/blog/[...slug]': '/articles/[...slug]',
+	 *   }
+	 * }
+	 * ```
+	 *
+	 *  
+	 * For statically-generated sites with no adapter installed, this will produce a client redirect using a [`<meta http-equiv="refresh">` tag](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meta#http-equiv) and does not support status codes.
+	 *
+	 * When using SSR or with a static adapter in `output: static`
+	 * mode (Netlify, Cloudflare, and Vercel), status codes are supported.
```

Comment:
```suggestion
	 * mode, status codes are supported.
```
I think we decided to remove this list.